cypress/page/products-list-page.ts

```diff
@@ -1,19 +1,26 @@
 class ProductsListPage {
   private productList: string;
   private proceedToCheckoutButton: string;
+  private productos: string;
 
   constructor() {
-    this.productList =
-      "a[title='Add to cart']";
-      this.proceedToCheckoutButton =".button-container a[title='Proceed to checkout']"
+    this.productList = "a[title='Add to cart']";
```

Comment:
The name `productList` is confusing, since it's a selector for an _Add to Cart button_